### PR TITLE
Added player pet

### PR DESCRIPTION
**Pull request by LilacGooseberries**
added player pet, added stun from warlock resummoning new pet (and some minor changes in defaults)

### CODE REVIEW

File: LoseControl.lua [hunk 1]
```diff
@@ -257,20 +261,26 @@ local DBdefaults = {
...
-		Root    = false,
+		Root    = true,
 		Snare   = false,
 	},
 	frames = {
 		player = {
 			enabled = true,
-			size = 48,
-			alpha = 0.65,
+			size = 36,
```
Comment (vendethiel) [inline LoseControl.lua:270]: any reason for changing the defaults?

**Comment by vendethiel** [comment]
also, fixes #5

File: localization.lua [hunk 2]
```diff
@@ -78,6 +80,7 @@ elseif locale == "frFR" then -- translated by Adirelle
 		["Immune"]  = "Immunisé",
 		["PvE"]     = "PvE",
 		["player"] = "Joueur",
+		["pet"]    = "Animal",
```
Comment (vendethiel) [inline localization.lua:83]: Is that from google translate? I'll change it later

File: LoseControl.lua (hunk 1 above)
Comment (LilacGooseberries) [reply LoseControl.lua:270]: no, changed it for myself

File: localization.lua (hunk 2 above)
Comment (LilacGooseberries) [reply localization.lua:83]: KEKW

**Review by vendethiel — APPROVED** [review]

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**